WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -7,6 +7,8 @@ struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .addProductVariations:
             return buildConfig == .localDeveloper || buildConfig == .alpha
+        case .editProductVariationAttributes:
+            return buildConfig == .localDeveloper || buildConfig == .alpha
```

Comment:
Thinking about it, I think we can reuse the `addProductVariations` feature flag(maybe rename it to `productVariations`)

Since this is the only place where the new feature flag will be used I think it's safe to use the existing one and then remove the conditional if we want to release the "edit" functionality first.

Reply:
> I think it's safe to use the existing one and then remove the conditional if we want to release the "edit" functionality first.

Ah, good point! I wanted to make it simple for releasing the edit functionality separately, but this also makes sense.

Reply:
I went ahead and reused the `addProductVariations` feature flag, but without renaming it. (I started to rename it, but it felt a little confusing in context since not all of the product variations feature is behind the flag.)

I left a comment on the conditional where it's used for the "edit" functionality, as a reminder if we want to release that before everything else.